subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -369,7 +367,7 @@ static int update_attrs(void *ref, struct notification_attrs *out)
 	return 0;
 }
 
-static void clear_attrs(void *ref)
+void clear_attrs(void *ref)
```

Comment:
Please place the declaration into _lwm2m_engine.h_ and document it properly. This same advice applies to all functions which are used from _lwm2m_registry.c_.

Reply:
So, every function I've changed from static to non-static should be in the lwm2m_engine.h?